lua/kubectl/views/pods/init.lua

```diff
@@ -9,6 +9,8 @@ local M = {
   builder = nil,
   selection = {},
   pfs = {},
+  is_tailing = false,
+  tail_handle = nil,
```

Comment:
I think you can improve the code a little bit byt using the handle as a check instead of having two variables, it's not much but a small improvement.
you would do something like this:
```
      if handle and not handle:is_closing() then
        handle:kill(2)
        handle = nil
      else
        handle = start_tailing
      end

```

Reply:
fixed.